src/support/z_sanitize.erl

```diff
@@ -296,6 +296,7 @@ preferred_protocol(<<"www.youtube.", _/binary>>) -> <<>>;
 preferred_protocol(<<"vimeo.", _/binary>>) -> <<>>;
 preferred_protocol(<<"www.vimeo.", _/binary>>) -> <<>>;
 preferred_protocol(<<"platform.instagram.com", _/binary>>) -> <<>>;
+preferred_protocol(<<"platform.twitter.com", _/binary>>) -> <<>>;
```

Comment:
Yeah, good one! Let’s do that change separately from this PR (as it applies to other services as well).